app/components/filter-list-widget/filter-list-widget-directive.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+angular.module('filterListWidget.directive', [
+  'bulbsCmsApp.settings'
+])
+  .directive('filterListWidget', function (_, $http, $location, $timeout, $, routes) {
+    return {
```

Comment:
Did you create this file?

If I'm understanding what the logic of this directive is, I would prefer that the [autocomplete-basic directive](https://github.com/theonion/bulbs-cms/blob/master/app/components/autocomplete-basic/autocomplete-basic.js) be used instead of creating a new directive.